lms/lmsweb/forms/change_password.py

```diff
@@ -0,0 +1,27 @@
+from flask_babel import gettext as _  # type: ignore
+from flask_wtf import FlaskForm
+from wtforms import PasswordField
+from wtforms.validators import EqualTo, InputRequired, Length, ValidationError
+
+
+class ChangePasswordForm(FlaskForm):
+    current_password = PasswordField(
+        'Password', validators=[InputRequired(), Length(min=8)], id='password',
+    )
+    password = PasswordField(
+        'Password', validators=[InputRequired(), Length(min=8)], id='password',
+    )
+    confirm = PasswordField(
+        'Password Confirmation', validators=[
+            InputRequired(),
+            EqualTo('password', message=_('הסיסמאות שהוקלדו אינן זהות')),
+        ],
+    )
+
+    def __init__(self, user, *args, **kwargs):
+        super(ChangePasswordForm, self).__init__(*args, **kwargs)
+        self.user = user
+
+    def validate_current_password(self, field):
```

Comment:
We must somehow put rate limiting on `is_password_valid` to prevent people exploiting it to reveal the true user's password